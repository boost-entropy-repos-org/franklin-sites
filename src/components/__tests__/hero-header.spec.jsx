import React from 'react';
import renderer from 'react-test-renderer';
import HeroHeader from '../hero-header';

describe('Hero Header component', () => {
  test('should render', () => {
    const component = renderer.create(
      <HeroHeader title="Title" footer={<span>Footer</span>}>
        <input type="text" />
      </HeroHeader>
    );
    expect(component).toMatchSnapshot();
  });
});
