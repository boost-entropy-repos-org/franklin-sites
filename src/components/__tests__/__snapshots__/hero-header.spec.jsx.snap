// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Header component should render 1`] = `
<div
  className="hero-header"
>
  <div
    className="hero-header__content"
  >
    <h1>
      Title
    </h1>
    <input
      type="text"
    />
  </div>
  <div
    className="hero-header__footer"
  >
    <span>
      Footer
    </span>
  </div>
</div>
`;
